client/jetpack-connect/style.scss

```diff
@@ -588,6 +588,21 @@ $colophon-height: 50px; // wpcomColophon element at the bottom
 	color: transparent;
 }
 
+.jetpack-connect__sso {
+
+	.formatted-header__title {
+		margin-bottom: 5px;
```

Comment:
Let's use known design system spatial dimensions here:

```suggestion
		margin-bottom: 8px;
```

Reply:
On larger screens, `5px` is used already. This change just maintains that on mobile. If we change to `8px` then we will need to update larger screens too.

Reply:
I see, let's keep the rework minimal then.